CHANGELOG.md

```diff
@@ -37,6 +37,12 @@
 
   ([PR #1054](https://github.com/alphagov/govuk-frontend/pull/1054))
 
+- Allow form group classes on date, file upload, input, select and textarea
```

Comment:
These will be new features not fixes, since they don't exist yet.

Reply:
Moved to new features, thanks! Sorry these changelogs get me every time